ayon_api/_api.py

```diff
@@ -4028,7 +4029,10 @@ def delete_link(*args, **kwargs):
 def get_entities_links(*args, **kwargs):
     """Helper method to get links from server for entity types.
 
-    Example output::
+    .. highlight:: text
+    .. code-block:: text
+
+        Example output
```

Comment:
Should this be without colon `:` (differs from docstring in `ServerAPI`)?

Reply:
Thanks for the tip, and sorry for the back and forth.